src/components/options/SpcOptionEditor.tsx

```diff
@@ -97,6 +105,13 @@ export function SpcOptionEditor(props: Props) {
           />
         </InlineField>
       </div>
+      {options.sampleSize > 10 ? (
```

Comment:
It seems like "magic number". Please make a constant or calculate its value based on  `props.value` or `options`.